src/main/java/executor/service/service/di/InjectByTypeAnnotationObjectConfigurator.java

```diff
@@ -0,0 +1,58 @@
+package executor.service.service.di;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.Parameter;
+
+public class InjectByTypeAnnotationObjectConfigurator implements ObjectConfigurator {
+
+    @Override
+    public void configure(Object t, ApplicationContext context) {
+        for (Field field : t.getClass().getDeclaredFields()) {
+            if (isFieldNotPrimitiveOrWrapper(field) &&
+                    isFieldInjectable(t, field)) {
+                field.setAccessible(true);
+                Object object = context.getObject(field.getType());
+                try {
+                    field.set(t, object);
+                } catch (IllegalAccessException e) {
+                    throw new RuntimeException(e);
+                }
+            }
+        }
+    }
+
+    private boolean isFieldNotPrimitiveOrWrapper(Field field) {
+        Class<?> fieldType = field.getType();
+        return !fieldType.isPrimitive()
+                && fieldType != String.class
+                && fieldType != Boolean.class
+                && fieldType != Character.class
+                && fieldType != Byte.class
+                && fieldType != Short.class
+                && fieldType != Integer.class
+                && fieldType != Long.class
+                && fieldType != Float.class
+                && fieldType != Double.class
+                && fieldType != Void.class;
+    }
```

Comment:
`
or 
` return !fieldType.isPrimitive() && !fieldType.getName().startsWith(excludedPackage);`
```suggestion
private boolean isFieldNotPrimitiveOrWrapper(Field field) {
        Class<?> fieldType = field.getType();
    List<Class<?>> excludedClasses = Arrays.asList(
        String.class, Boolean.class, Character.class,
        Byte.class, Short.class, Integer.class,
  Long.class, Float.class, Double.class, Void.class
);
return !fieldType.isPrimitive() && !excludedClasses.contains(fieldType);
//or
private boolean isFieldNotPrimitiveOrWrapper(Field field) {
        Class<?> fieldType = field.getType();
String excludedPackage = "java.lang";
return !fieldType.isPrimitive() && !excludedClasses.contains(fieldType);
```

Reply:
I don't mind, we can replace